content/news/052/index.md

```diff
@@ -65,7 +65,16 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Announcements
 
-TODO: I'll be working on adding an email subscription this weekend, so I'll write about it here once that's done :) - Jan
+Since years, our readers have asked for a way to subscribe to this newsletter by email. This again came up in our [recent survey][survey].
+We're happy to announce that this feature is now available! When you visit our [homepage],
+you can now scroll a bit down and find an email subscription form. 
```

Comment:
nitpick as I feel it's not too important information

```suggestion
you can now scroll down and find an email subscription form. 
```

or maybe that's what was bugging me 🤔 
```suggestion
you can now scroll down a bit and find an email subscription form. 
```